### ISSUE TITLE
Cannot download videos with an ID over 2147483647

### ISSUE THREAD

**Issue by KoenDG** [ihabunek/twitch-dl#157]
For those not aware: that number is the maximum value of an integer.

Tested with a value just below and just above. Behavior confirmed.

Example error output:

```
GraphQL query failed
* Argument "id" has invalid value 2147633472.
Expected type "ID", found 2147633472.
```

Tested with latest version.

**Comment by ihabunek** [comment]
This was fixed in version 2.3.1 which was released earlier today, please upgrade.

**Comment by KoenDG** [comment]
Strange, pretty sure I tested it with the latest version. Must have missed something on my end, it's working now.
